### ISSUE TITLE
More Atmosphere types[Request]

### ISSUE THREAD

**Issue by AGiLE-EaGLE1994** [zmaster587/AdvancedRocketry#1584]
**What do you think is missing or what do you think can be done better**
[Describe in detail what you want to see and how it makes the mod better]
Might I suggest more types of atmospheres? Such as:
Polluted atmospheres (Which would cause  visual FX when entered without a spacesuit or exiting from a building with a sealed oxygen system, and give poison damage for the first few minutes of exposure,  nausea after 10 minutes, and Wither plus nausea after 15 minutes. Polluted atmospheres could be survived by wearing a spacesuit, helmet with Respiration or Water Breathing, or entering a building which has its own oxygen system as if it were a space station.)
Acidic atmospheres, which would cause rain to be harmful when it touched you, and cause pools of acid to form which you would be hurt by if you walked into it. (Acidic atmospheres can be combatted by wearing a spacesuit)
Extreme high pressure atmospheres (Which would squash you like an anvil when you aren't wearing a pressure suit or in creative mode.

**Comment by MrEyeballs29** [comment]
Is that a duplicate from another issue.

**Comment by AGiLE-EaGLE1994** [comment]
I don't believe so. I specifically asked for a polluted atmosphere.

**Comment by MrEyeballs29** [comment]
I already posted an issue where there should be more types of atmospheres
instead of normal boring oxygen. Even high pressure is not realistic
because it slows you down instead of damaging.

**Comment by AGiLE-EaGLE1994** [comment]
Yeah, but we asked for different types, so I don't think it counts as a duplicate. I was wanting polluted atmospheres.
